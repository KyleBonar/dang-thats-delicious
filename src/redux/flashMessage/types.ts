// Declare state types with `readonly` modifier to get compile time immutability.
// https://github.com/piotrwitek/react-redux-typescript-guide#state-with-type-level-immutability
export interface IFlashMessage {
  readonly isVisible: boolean;
  readonly type: null | string;
  readonly text: null | string;
  readonly slug: null | string;
}
