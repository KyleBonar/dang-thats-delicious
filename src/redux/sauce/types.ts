// Declare state types with `readonly` modifier to get compile time immutability.
// https://github.com/piotrwitek/react-redux-typescript-guide#state-with-type-level-immutability
export interface ISauce {
  _id: number;
  name: string;
  ingredients: string;
  types?: string[];
  maker: string;
  description: string;
  photo?: string;
  shu?: number | string;
  peppers?: string[];
  reviews?: number[];
  location?: {
    city?: string;
    state?: string;
    country?: string;
  };
  tags?: string[];
}

// Trimmed down for reference only
export interface ISauceRef {
  _id: number;
}
