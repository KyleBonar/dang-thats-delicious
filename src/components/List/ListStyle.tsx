import styled from "../../theme/styled-components";
import { Link } from "../Link/Link";

export const StyledDiv = styled.div`
  padding: 10px;
`;
StyledDiv.displayName = "div";

export const StyledH5 = styled.h5`
  margin: 0 0 0.5em 0;
  font-weight: 400;
  padding: 0;
  text-transform: uppercase;
  font-size: ${props => props.theme.scaleH5};
`;
StyledH5.displayName = "h5";

export const StyledUl = styled.ul`
  margin: 0px;
  padding: 0px;
  list-style: none;

  li {
    margin-top: 0.75rem;
    margin-bottom: 0.75rem;
    padding-left: 0.75rem;
  }

  @media (min-width: ${props => props.theme.smToMd}) {
    li {
      padding-left: 0;
    }
  }
`;
StyledUl.displayName = "ul";
