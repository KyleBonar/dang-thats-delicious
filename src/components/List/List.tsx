import * as React from "react";
import * as shortid from "shortid";
import { Link } from "../Link/Link";

import { StyledDiv, StyledH5, StyledUl } from "./ListStyle";

interface ListProps {
  className?: string;
  title: string;
  items: Array<{
    link?: string;
    text: string;
  }>;
}

const List: React.FunctionComponent<ListProps> = props => {
  return (
    <StyledDiv className={props.className}>
      <StyledH5>{props.title}</StyledH5>
      <StyledUl>
        {props.items.map(item => {
          return (
            <li key={shortid.generate()}>
              <Link href={item.link || "#"}>{item.text}</Link>
            </li>
          );
        })}
      </StyledUl>
    </StyledDiv>
  );
};

export default List;
