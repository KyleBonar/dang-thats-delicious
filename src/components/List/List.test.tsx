import * as React from "react";
import * as enzyme from "enzyme";
import List from "./List";

import { fakeLists } from "../../utils/testUtils/testUtils";
import { MockList } from "../../utils/testUtils/types";

const mockLists: MockList[] = fakeLists();

describe("<List />", () => {
  it("renders", () => {
    mockLists.forEach((mockList: MockList) => {
      const wrapper = enzyme.shallow(<List {...mockList}></List>);

      expect(wrapper).toBeTruthy();
    });
  });

  it("matches snapshot", () => {
    mockLists.forEach((mockList: MockList) => {
      const wrapper = enzyme.shallow(<List {...mockList}></List>);

      expect(wrapper).toMatchSnapshot();
    });
  });

  it("renders correct title", () => {
    mockLists.forEach((mockList: MockList) => {
      const wrapper = enzyme.shallow(<List {...mockList}></List>);

      expect(wrapper.find("h5").text()).toEqual(mockList.title);
    });
  });

  it("renders correct number of list items", () => {
    mockLists.forEach((mockList: MockList) => {
      const wrapper = enzyme.shallow(<List {...mockList}></List>);

      expect(wrapper.find("li").length).toEqual(mockList.items.length);
    });
  });

  // it("renders correct list item texts", () => {
  //   wrapper.find("li").map((ind, node) => {
  //     // li -> a -> text -> data (same as text())
  //     expect(node.children[0].children[0].data).toEqual(items[ind].text);
  //   });
  // });

  // it("renders correct list item link", () => {
  //   wrapper.find("li").map((ind, node) => {
  //     // li -> a
  //     expect(node.children[0].attribs.href).toEqual(items[ind].link || "#");
  //   });
  // });
});
