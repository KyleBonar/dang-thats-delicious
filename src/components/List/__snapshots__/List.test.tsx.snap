// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<List /> matches snapshot 1`] = `
<StyledDiv>
  <StyledH5>
    sequi facilis corporis sed veritatis quidem praesentium
  </StyledH5>
  <StyledUl>
    <li
      key="VXuEXXZRY2q"
    >
      <Styled(LinkComponent)
        href="http://Walter.name/"
      >
        Assumenda aut enim iure facilis ut explicabo iure. Occaecati officia iure nisi. Maiores recusandae fuga reprehenderit soluta non omnis consequatur enim quia. Et modi tenetur amet modi reprehenderit. Adipisci eius laborum eius aut illo dolorum.
      </Styled(LinkComponent)>
    </li>
    <li
      key="EPA1BqhQEWd"
    >
      <Styled(LinkComponent)
        href="http://www.Lelah.us/"
      >
        Ipsa tempora aut sint aperiam occaecati illum animi vel. Dicta perferendis dolorum cupiditate illum saepe ut. Sint quia sed exercitationem omnis laborum exercitationem est. Laudantium molestias assumenda quasi adipisci mollitia iste itaque omnis debitis. Similique temporibus aliquam sed sapiente soluta. Commodi eaque nisi cum ducimus.
      </Styled(LinkComponent)>
    </li>
    <li
      key="51LmHXbg29g"
    >
      <Styled(LinkComponent)
        href="http://www.Yazmin.biz/"
      >
        Non explicabo dolores ipsam laboriosam non itaque voluptas ea. Autem sit aspernatur aspernatur quam eligendi. Fugiat dolorem nihil quam nulla est unde sit. Expedita est doloremque explicabo eaque.
      </Styled(LinkComponent)>
    </li>
    <li
      key="Rm5KsYWU2Qu"
    >
      <Styled(LinkComponent)
        href="http://Ethelyn.org/"
      >
        Ullam quae corporis qui aut nobis quo consequatur harum qui. Et doloremque voluptate in sunt. Nulla aut sunt est rerum at sint. Fugiat quis et voluptas. Et vero repellat eveniet assumenda quis quae. Sapiente nostrum aut corrupti sed consectetur.
      </Styled(LinkComponent)>
    </li>
    <li
      key="Xg30kf3ptsn"
    >
      <Styled(LinkComponent)
        href="http://Jamel.us/"
      >
        Qui dignissimos dolore qui voluptatem cumque nihil iste est. Rem similique et maxime quia dolores quisquam. Consequatur sint provident aut. Distinctio autem quam qui facere. Ut odit animi in ut occaecati. Odit tempora at modi occaecati illum laborum quos eum.
      </Styled(LinkComponent)>
    </li>
    <li
      key="BFmAwV_bZtN"
    >
      <Styled(LinkComponent)
        href="http://www.Sofia.org/"
      >
        Itaque et in fugit in rerum libero odit aut. Qui aut inventore aut eveniet. Incidunt ut ducimus fugiat. Et aut quidem voluptatibus libero aut autem. Ullam et possimus sint fugiat maiores autem est esse quos.
      </Styled(LinkComponent)>
    </li>
  </StyledUl>
</StyledDiv>
`;

exports[`<List /> matches snapshot 2`] = `
<StyledDiv>
  <StyledH5>
    minus deserunt nihil quasi minus quo sunt
  </StyledH5>
  <StyledUl>
    <li
      key="5gdPPyPJg8n"
    >
      <Styled(LinkComponent)
        href="http://Grady.io/"
      >
        Consequatur velit voluptate necessitatibus excepturi et qui sunt. Provident eaque atque quos. Nobis nam dolor quas repellendus in dolores totam animi ipsum. Et consequatur consectetur sunt molestiae cupiditate autem. Consequatur accusantium molestiae.
      </Styled(LinkComponent)>
    </li>
    <li
      key="q0bEEuMmCp9"
    >
      <Styled(LinkComponent)
        href="http://Bergstrom.co.uk/"
      >
        Dolorem quia consequatur omnis ab assumenda animi. Quis quis nobis repudiandae aliquam molestiae occaecati. Culpa facilis eligendi voluptatibus similique blanditiis in eum ut dignissimos.
      </Styled(LinkComponent)>
    </li>
    <li
      key="KXrr9kAhgHO"
    >
      <Styled(LinkComponent)
        href="http://www.Camilla.name/"
      >
        Amet quia quia aut. Cumque sunt mollitia quibusdam officia eum repudiandae minima temporibus. Quia assumenda ut neque saepe est. Sed rerum nesciunt voluptas rem.
      </Styled(LinkComponent)>
    </li>
    <li
      key="7NkdQmWbKBV"
    >
      <Styled(LinkComponent)
        href="http://Volkman.ca/"
      >
        Ut corrupti voluptatem quia. Dolor voluptates sequi perferendis nostrum aliquid. Animi architecto accusantium vel enim sint. Porro ut est sed qui.
      </Styled(LinkComponent)>
    </li>
    <li
      key="hOKwgU_Q5dB"
    >
      <Styled(LinkComponent)
        href="http://Kunde.net/"
      >
        Molestias autem expedita omnis sit vitae consequuntur non. Et ut voluptatem quia. Eum velit aperiam voluptas.
      </Styled(LinkComponent)>
    </li>
    <li
      key="PXll4s8RtPT"
    >
      <Styled(LinkComponent)
        href="http://www.Rahsaan.ca/"
      >
        Excepturi saepe autem quis molestiae et dicta odit. Quisquam qui harum a aut esse unde ea voluptatem. Occaecati eveniet aut eligendi iusto sed. Voluptates odio rem saepe iure quia culpa suscipit totam. Necessitatibus quis ratione eveniet ea totam aut aut.
      </Styled(LinkComponent)>
    </li>
    <li
      key="m0NbQmvGx-a"
    >
      <Styled(LinkComponent)
        href="http://Jany.biz/"
      >
        Quibusdam quasi quisquam ut libero. At quae autem eligendi. Quis eveniet sit voluptas dicta non dolore autem. Optio est eius molestiae. Architecto ut quasi quasi accusantium modi. Rerum non quis sed ea qui.
      </Styled(LinkComponent)>
    </li>
    <li
      key="Ed8G8g2mAwo"
    >
      <Styled(LinkComponent)
        href="http://www.Margarita.me/"
      >
        Eos commodi at impedit sint. In dolorem repudiandae. Dignissimos esse qui est aut laborum nesciunt dolor. Voluptate esse aliquam molestiae quas omnis qui illo excepturi. Porro impedit omnis quo aut quia. Architecto eos quos tenetur voluptas quia nam laborum.
      </Styled(LinkComponent)>
    </li>
    <li
      key="GC4rWNeoMvG"
    >
      <Styled(LinkComponent)
        href="http://Jeramy.com/"
      >
        Perspiciatis ut illo sit rerum accusamus molestiae quia est reprehenderit. Perspiciatis quas et quos esse dolores. Delectus molestias ut distinctio. Placeat exercitationem unde minus facere iste sunt. Aut dolorem in non dolorem.
      </Styled(LinkComponent)>
    </li>
    <li
      key="VHtssoM2a6X"
    >
      <Styled(LinkComponent)
        href="http://Powlowski.info/"
      >
        Sit laboriosam quis voluptatem quod tempore. Impedit dolores saepe dolor incidunt vero quia voluptatibus consequatur. Qui delectus qui qui ab officiis omnis.
      </Styled(LinkComponent)>
    </li>
    <li
      key="obsNtpgLtL3"
    >
      <Styled(LinkComponent)
        href="http://www.Bergnaum.org/"
      >
        Officia aut id possimus quas dolor error eum. Provident deleniti consequuntur velit ad. Omnis fugit labore quia et beatae adipisci voluptatum id porro. Numquam consequatur et et voluptas repellendus.
      </Styled(LinkComponent)>
    </li>
    <li
      key="luYswt8_Oll"
    >
      <Styled(LinkComponent)
        href="http://www.Joana.io/"
      >
        Sed illum et ea quia sunt est aut et. Libero et et ratione qui voluptate sit. Sunt consectetur perspiciatis nisi repellendus quia voluptate. Suscipit delectus aliquid et dicta ipsam qui provident. Necessitatibus blanditiis deserunt accusantium debitis aut laborum at non. Sit officia doloremque non sit a perferendis soluta dignissimos.
      </Styled(LinkComponent)>
    </li>
    <li
      key="nhQLuY7ygMK"
    >
      <Styled(LinkComponent)
        href="http://Hermiston.us/"
      >
        Sapiente molestiae dolores voluptas placeat vitae ut. Recusandae ea necessitatibus sed ea ab doloremque est. Iste omnis eaque nobis possimus. Quia quam sit optio rerum quasi occaecati modi soluta. Omnis atque inventore vitae sint eligendi quis excepturi veritatis occaecati.
      </Styled(LinkComponent)>
    </li>
    <li
      key="B99oWhm8qgt"
    >
      <Styled(LinkComponent)
        href="http://www.Sadye.com/"
      >
        Veritatis quisquam eius dolore quia error sed asperiores amet. Quos repellat distinctio libero mollitia facilis est laboriosam tempora. Cupiditate aut non ipsam fuga rem eius est nihil. Enim itaque nostrum omnis non.
      </Styled(LinkComponent)>
    </li>
    <li
      key="OIj9H1dIVpH"
    >
      <Styled(LinkComponent)
        href="http://Enrico.io/"
      >
        Iure dicta ab minus animi quis. Animi aut sapiente fuga unde aliquam et. Dolorem possimus accusantium enim est aspernatur voluptatem consequatur temporibus. Numquam ut et. Cupiditate minus cum est.
      </Styled(LinkComponent)>
    </li>
    <li
      key="kGKGPR5CdN_"
    >
      <Styled(LinkComponent)
        href="http://Reuben.ca/"
      >
        Et sint architecto corrupti quia nostrum. Quae non doloremque voluptas repellendus voluptatum. Ut dolor maiores aut quasi quia. Quia blanditiis qui sed a dolores voluptatem qui.
      </Styled(LinkComponent)>
    </li>
    <li
      key="weTpypDKjG2"
    >
      <Styled(LinkComponent)
        href="http://www.Gabriel.org/"
      >
        Vel earum quo fugiat et ut. Delectus officia maiores cumque consequuntur rerum. Nulla aut omnis blanditiis. Ut odit quo enim facere illo nemo iusto iste. Dolor reprehenderit architecto. Laborum libero odio modi consequuntur et.
      </Styled(LinkComponent)>
    </li>
    <li
      key="PcX1KBN0Bho"
    >
      <Styled(LinkComponent)
        href="http://Mosciski.org/"
      >
        Ut eum doloribus. Perspiciatis sunt eligendi nihil porro suscipit sit eos maxime. Doloribus aut maiores laborum soluta rerum atque velit voluptatem. Quia accusantium autem consequuntur qui repellat laudantium et qui et.
      </Styled(LinkComponent)>
    </li>
    <li
      key="yaQZIevZ5mk"
    >
      <Styled(LinkComponent)
        href="http://www.Lynn.tv/"
      >
        Et id qui. Vel quaerat labore rerum aut voluptatem. Aliquam et ipsam dolores voluptatem et autem doloremque earum cum. Perferendis id dolor eum ut quisquam voluptatum.
      </Styled(LinkComponent)>
    </li>
  </StyledUl>
</StyledDiv>
`;

exports[`<List /> matches snapshot 3`] = `
<StyledDiv>
  <StyledH5>
    qui exercitationem eum laudantium autem et recusandae
  </StyledH5>
  <StyledUl>
    <li
      key="ThraxR5oJ7n"
    >
      <Styled(LinkComponent)
        href="http://Alice.io/"
      >
        Et optio laborum officia. Eius veritatis voluptates suscipit reiciendis in. Et error tempore aspernatur soluta. Voluptas sed esse rem sed repellendus quaerat similique. Doloribus et voluptatem id quisquam aspernatur quis consequatur eos.
      </Styled(LinkComponent)>
    </li>
    <li
      key="4fa1xW1n03f"
    >
      <Styled(LinkComponent)
        href="http://www.Stroman.org/"
      >
        Officiis assumenda nihil ad ut maxime aliquid. Voluptas saepe dignissimos voluptas quod omnis rerum. Ipsa qui voluptas sed. Officia aut alias aut aut nihil. Dolores voluptas quidem excepturi temporibus. Ut velit odit ratione saepe id officia pariatur.
      </Styled(LinkComponent)>
    </li>
    <li
      key="55o8volf1he"
    >
      <Styled(LinkComponent)
        href="http://Hettinger.net/"
      >
        Libero quas in ullam. Minima consequatur eligendi excepturi fugit beatae ut ea rerum nobis. Nihil sint occaecati.
      </Styled(LinkComponent)>
    </li>
    <li
      key="yuoUAq9Tojp"
    >
      <Styled(LinkComponent)
        href="http://www.Lockman.us/"
      >
        Ex nisi repudiandae qui ex unde. Soluta qui adipisci vero dolore. Iste eos perspiciatis. Quod a nisi et est. Quia ab sed. Ullam animi excepturi laborum veniam asperiores et consectetur animi.
      </Styled(LinkComponent)>
    </li>
    <li
      key="7qXyy6PCt0c"
    >
      <Styled(LinkComponent)
        href="http://Hoppe.info/"
      >
        Ratione sed enim provident. Qui odit nam doloribus et. Voluptatibus quas consequuntur dolores nihil eligendi sunt aliquid. Aliquid reiciendis dolorem porro voluptates ut amet ea repellendus. Est quis est veniam eum laudantium accusamus.
      </Styled(LinkComponent)>
    </li>
    <li
      key="RA9rT3jQhdy"
    >
      <Styled(LinkComponent)
        href="http://Douglas.info/"
      >
        Ut dolor et placeat rerum. Ex adipisci fugiat officiis. Voluptatem adipisci sequi magnam doloremque. Dolorum magni molestias dolorem nesciunt sed.
      </Styled(LinkComponent)>
    </li>
    <li
      key="Z5NDVoOZUH4"
    >
      <Styled(LinkComponent)
        href="http://www.Ebert.name/"
      >
        Iure iste aut ab laborum sint qui id ipsa est. Magni ea laudantium hic quo corporis minima cupiditate. Voluptatibus officiis voluptatem veniam. Voluptas suscipit sit et culpa cum architecto nulla ut.
      </Styled(LinkComponent)>
    </li>
    <li
      key="U4qAtFpAfDl"
    >
      <Styled(LinkComponent)
        href="http://Elisha.biz/"
      >
        Incidunt porro amet at asperiores libero officia totam corporis rem. Eligendi ipsa molestiae aut doloribus. Sunt dolorum tempore animi vel possimus eos. Ducimus voluptatem iusto tempore voluptas. Accusamus vitae eligendi et aperiam quia unde ex.
      </Styled(LinkComponent)>
    </li>
  </StyledUl>
</StyledDiv>
`;

exports[`<List /> matches snapshot 4`] = `
<StyledDiv>
  <StyledH5>
    magni ullam ullam doloremque quidem vel voluptatem
  </StyledH5>
  <StyledUl>
    <li
      key="OBGcQVkXAco"
    >
      <Styled(LinkComponent)
        href="http://www.Devonte.biz/"
      >
        Laboriosam sed est. Sapiente ipsa aut nostrum repudiandae harum neque ut dolores. Qui architecto aliquid iure eos expedita assumenda. Aut nostrum natus ullam voluptatem accusantium blanditiis iste.
      </Styled(LinkComponent)>
    </li>
  </StyledUl>
</StyledDiv>
`;

exports[`<List /> matches snapshot 5`] = `
<StyledDiv>
  <StyledH5>
    animi ipsa est molestias deserunt quo impedit
  </StyledH5>
  <StyledUl>
    <li
      key="tjaYQ1uM1sB"
    >
      <Styled(LinkComponent)
        href="http://Jany.biz/"
      >
        Impedit assumenda qui cum quos eos iste aut. Optio tempora excepturi. Molestiae quae ut dolores quam. Dolores tempore facilis rerum unde minima.
      </Styled(LinkComponent)>
    </li>
    <li
      key="pT_CB1PYI2C"
    >
      <Styled(LinkComponent)
        href="http://www.Heaney.me/"
      >
        Voluptatem nam in quod et. Quasi perferendis voluptatem nemo consequatur voluptatum. Eos magni iste doloribus et fuga adipisci saepe. Quia veritatis qui optio autem et. Explicabo provident molestiae qui est laudantium corporis.
      </Styled(LinkComponent)>
    </li>
    <li
      key="YmJhN0Ud3IA"
    >
      <Styled(LinkComponent)
        href="http://www.Leonie.info/"
      >
        Vel omnis qui qui necessitatibus. Tempora voluptatem autem. Delectus iure veniam est consequatur ut et numquam. Culpa nulla ipsum et explicabo vel quaerat fuga enim. Esse quia aperiam libero.
      </Styled(LinkComponent)>
    </li>
    <li
      key="6UtTg7p42ab"
    >
      <Styled(LinkComponent)
        href="http://www.Funk.tv/"
      >
        Unde quia incidunt ex et molestiae facere. Et ut unde consequatur sequi. Reiciendis blanditiis vel nisi culpa provident voluptatem sit id. Tenetur iure quos sed quibusdam voluptatem ducimus. Vel quia fugit totam quia esse quod aut. Rem sunt aut quam provident voluptates perspiciatis nulla maiores pariatur.
      </Styled(LinkComponent)>
    </li>
    <li
      key="QlGzR0vQhV3"
    >
      <Styled(LinkComponent)
        href="http://www.Rogahn.me/"
      >
        Vero dolores incidunt est nostrum quae et. Dolor doloribus cum qui aut natus aperiam est est. Aut adipisci ab ut unde. Beatae similique totam quia eum. Praesentium ducimus facere esse culpa aspernatur earum ex officia.
      </Styled(LinkComponent)>
    </li>
    <li
      key="7RfWHk-o0-e"
    >
      <Styled(LinkComponent)
        href="http://www.Shawn.biz/"
      >
        Dicta sit cum voluptatem dolor delectus unde reprehenderit. Nihil ex maiores nihil quisquam id. Sint voluptatum cupiditate quo sit adipisci ab rerum. Consequatur assumenda odio id id repellat. Iure fuga sit quod officiis exercitationem ut expedita quos nam.
      </Styled(LinkComponent)>
    </li>
    <li
      key="cg5qOoz5OlJ"
    >
      <Styled(LinkComponent)
        href="http://www.Tomas.me/"
      >
        Suscipit impedit veniam perferendis est aliquam consequuntur. Velit quibusdam aut in blanditiis quia aliquam. Exercitationem excepturi eligendi facilis similique optio hic perspiciatis dolor. Doloribus culpa possimus consectetur voluptas perferendis dicta iste. Earum et nam rem adipisci ut nisi a delectus.
      </Styled(LinkComponent)>
    </li>
    <li
      key="TQJs2WQPmk3"
    >
      <Styled(LinkComponent)
        href="http://Bogisich.com/"
      >
        Aut qui veniam quis sapiente laboriosam. Expedita quam ex et nobis saepe dicta. Asperiores nihil illum ut. Velit velit perspiciatis mollitia ut corrupti sunt temporibus. Consequatur porro excepturi aperiam qui voluptatem rerum et cumque.
      </Styled(LinkComponent)>
    </li>
    <li
      key="1jKooAmrwb6"
    >
      <Styled(LinkComponent)
        href="http://Schowalter.net/"
      >
        Aperiam voluptas impedit cumque. Quod et dolorem possimus quasi accusantium quaerat. Perferendis commodi harum et reiciendis. Quis dolor non id cum enim mollitia.
      </Styled(LinkComponent)>
    </li>
    <li
      key="iSMZVIdnXc6"
    >
      <Styled(LinkComponent)
        href="http://Murazik.name/"
      >
        Aut eaque dignissimos incidunt quis non. Molestiae repellendus quibusdam delectus non. Sit illo consequatur itaque vero repellendus ex. Possimus animi et sapiente qui minima ea beatae.
      </Styled(LinkComponent)>
    </li>
    <li
      key="XdPQbNhgM3o"
    >
      <Styled(LinkComponent)
        href="http://www.Grady.tv/"
      >
        Laboriosam molestiae modi expedita quaerat eos aut. Ut quis incidunt provident assumenda sit pariatur quam et excepturi. Sit aut iste explicabo expedita eos ut. Unde est doloremque et molestiae velit sequi sapiente. Voluptas at illum aspernatur qui et nam eaque sit. Dolor corrupti sit veniam est molestias odio qui.
      </Styled(LinkComponent)>
    </li>
    <li
      key="PpRC8dhM5nV"
    >
      <Styled(LinkComponent)
        href="http://www.Jacobs.com/"
      >
        Et quaerat expedita sunt est. Ut non fugit sint est dolores dolore. Ipsum ratione expedita modi. Nihil et voluptatem ut ipsa iure similique. Minima sunt deserunt sit aut et.
      </Styled(LinkComponent)>
    </li>
    <li
      key="yWTmI02Yisq"
    >
      <Styled(LinkComponent)
        href="http://www.Mohamed.biz/"
      >
        Rerum voluptas veniam ea veniam optio. Dolor exercitationem vitae magni a. Eveniet non minus eaque adipisci eos. Error velit modi. Sunt exercitationem aliquam. Quisquam vel saepe et qui aut enim.
      </Styled(LinkComponent)>
    </li>
  </StyledUl>
</StyledDiv>
`;

exports[`<List /> matches snapshot 6`] = `
<StyledDiv>
  <StyledH5>
    culpa eius possimus voluptatem nemo laboriosam est
  </StyledH5>
  <StyledUl>
    <li
      key="rWM7rZSTFui"
    >
      <Styled(LinkComponent)
        href="http://www.Emard.org/"
      >
        Dolor modi tenetur enim praesentium voluptatem ipsam quod. Molestiae temporibus non dolorum natus. Facere qui sint enim ratione cupiditate.
      </Styled(LinkComponent)>
    </li>
    <li
      key="s_4H0jgi9aq"
    >
      <Styled(LinkComponent)
        href="http://Lindsey.info/"
      >
        Dicta ipsum vel tenetur expedita. Pariatur magni ut. Iste quia fugiat ducimus. Sit eum possimus vel expedita. Labore dolores enim magni perspiciatis voluptas officia voluptatem qui.
      </Styled(LinkComponent)>
    </li>
  </StyledUl>
</StyledDiv>
`;

exports[`<List /> matches snapshot 7`] = `
<StyledDiv>
  <StyledH5>
    nihil mollitia suscipit quisquam aut velit dignissimos
  </StyledH5>
  <StyledUl>
    <li
      key="pjeM7EI-4eb"
    >
      <Styled(LinkComponent)
        href="http://www.Wolf.me/"
      >
        Sed totam corporis error quae laudantium quos. Et non quis illum. Nostrum voluptas quia nam quisquam. Nostrum rerum nobis harum.
      </Styled(LinkComponent)>
    </li>
    <li
      key="nbsHpPDJMhL"
    >
      <Styled(LinkComponent)
        href="http://Frami.org/"
      >
        Vel deserunt et facilis consequatur aut neque. Cum minus et consequatur veritatis animi autem necessitatibus occaecati. Ducimus nostrum nemo adipisci dolores. Iure modi quibusdam.
      </Styled(LinkComponent)>
    </li>
    <li
      key="UO-kPnY96eC"
    >
      <Styled(LinkComponent)
        href="http://www.Kling.net/"
      >
        Est voluptatem et perferendis. Eos perspiciatis autem quae repellat facilis praesentium. Officia voluptas omnis provident sed facere ducimus cum. Non ad fugiat molestias accusamus impedit sed. Vitae mollitia temporibus minus qui ea aliquam hic.
      </Styled(LinkComponent)>
    </li>
    <li
      key="lYuvuDtpaxc"
    >
      <Styled(LinkComponent)
        href="http://www.Batz.name/"
      >
        Ut deleniti ad quaerat. Et neque ipsum vero. Id non non eum accusamus rem totam impedit enim doloremque. Nulla est id corporis reiciendis laborum enim. Perferendis doloremque qui voluptatibus perspiciatis quia eveniet. Laborum voluptatem dolor rem est asperiores.
      </Styled(LinkComponent)>
    </li>
    <li
      key="mJ9YTBTwRtH"
    >
      <Styled(LinkComponent)
        href="http://www.Helena.us/"
      >
        Aut recusandae doloremque quaerat aut. Ipsum itaque sunt ipsam hic facilis. Veniam est minus velit error officia ea vel laborum. Nihil facilis eum blanditiis ut.
      </Styled(LinkComponent)>
    </li>
    <li
      key="ohxhmGQ-K_f"
    >
      <Styled(LinkComponent)
        href="http://www.Upton.biz/"
      >
        Nulla dolorem natus et quisquam aut rem corporis praesentium. Omnis rerum voluptatum quaerat nulla aperiam aut. Voluptatem molestias beatae ea sed perferendis.
      </Styled(LinkComponent)>
    </li>
    <li
      key="CwxsnBFJd3X"
    >
      <Styled(LinkComponent)
        href="http://www.Devonte.net/"
      >
        Et est laudantium rerum. Eum mollitia reiciendis ipsam id nisi. Accusantium non id aliquid.
      </Styled(LinkComponent)>
    </li>
    <li
      key="-cAc2yBIQ_W"
    >
      <Styled(LinkComponent)
        href="http://www.Velda.net/"
      >
        Autem ducimus cum neque tempore ipsa perspiciatis. Laudantium sed dignissimos. Sed adipisci dolor.
      </Styled(LinkComponent)>
    </li>
    <li
      key="VoOwkjF7MyG"
    >
      <Styled(LinkComponent)
        href="http://www.Curt.net/"
      >
        Est et porro tempora est aliquid iure. Officia in excepturi dolorum. Non necessitatibus blanditiis atque beatae voluptatibus et nobis facilis. Molestiae voluptatem id id laborum commodi.
      </Styled(LinkComponent)>
    </li>
    <li
      key="kBfgmcRpGrZ"
    >
      <Styled(LinkComponent)
        href="http://www.Will.com/"
      >
        Ab sit incidunt ea est. Ad facilis vitae sit soluta odit voluptas architecto est. Dicta iusto placeat et aut. In nisi quia.
      </Styled(LinkComponent)>
    </li>
    <li
      key="zzyPI5piVNv"
    >
      <Styled(LinkComponent)
        href="http://Haleigh.us/"
      >
        Eos rerum doloribus voluptas aliquid optio impedit omnis aut. Aut dolor soluta aspernatur. Qui optio et voluptatem. Architecto molestiae suscipit mollitia et non autem et. Labore rerum qui.
      </Styled(LinkComponent)>
    </li>
    <li
      key="ArLwkFTF5dX"
    >
      <Styled(LinkComponent)
        href="http://Adonis.biz/"
      >
        Deleniti quas ut ullam. Magnam ad quod eum cupiditate delectus sed. Non maiores nam est voluptas quod rem voluptas voluptatem. Beatae repellendus doloribus iusto sint.
      </Styled(LinkComponent)>
    </li>
    <li
      key="_gYnr0-CCAz"
    >
      <Styled(LinkComponent)
        href="http://West.us/"
      >
        Magnam est voluptatum autem et sequi sit. Nihil dolorum quia tenetur est. Dolorum non maxime modi aliquid. Mollitia tempore delectus eos sunt optio iusto consequatur perferendis. Libero esse inventore enim necessitatibus nihil autem.
      </Styled(LinkComponent)>
    </li>
    <li
      key="5nLORduXCN7"
    >
      <Styled(LinkComponent)
        href="http://Rosa.io/"
      >
        Dolorem non earum et. Voluptates veritatis reprehenderit aperiam doloribus nisi tempore eius omnis et. Quae ut veritatis. Reprehenderit explicabo ducimus maiores velit aut repellendus animi. Aliquid amet ab dolorem aut qui nulla consequatur.
      </Styled(LinkComponent)>
    </li>
    <li
      key="g7jYbGIfViw"
    >
      <Styled(LinkComponent)
        href="http://Ignatius.io/"
      >
        Laborum repudiandae quo sint architecto ut quisquam ipsam. Possimus dignissimos quia minima laborum voluptatem ut unde quidem. Repudiandae mollitia minus error placeat optio non. Et quae nostrum.
      </Styled(LinkComponent)>
    </li>
    <li
      key="ZV5QmQDOzKl"
    >
      <Styled(LinkComponent)
        href="http://General.tv/"
      >
        Quia perspiciatis accusantium ducimus. Omnis accusamus dolorum quia ut quos ipsa cum. Sit totam et amet voluptatum qui et nisi reprehenderit. Et debitis atque blanditiis.
      </Styled(LinkComponent)>
    </li>
    <li
      key="YmpNF2ibzUT"
    >
      <Styled(LinkComponent)
        href="http://www.Alexander.biz/"
      >
        Molestiae vero ab nulla et et porro dolorem autem eum. Cumque a quibusdam et. Cupiditate quis ad asperiores eum est. Quas exercitationem aperiam tempora delectus ut labore.
      </Styled(LinkComponent)>
    </li>
  </StyledUl>
</StyledDiv>
`;

exports[`<List /> matches snapshot 8`] = `
<StyledDiv>
  <StyledH5>
    facere eius sint est dolores magni est
  </StyledH5>
  <StyledUl>
    <li
      key="DUceKLyyxZ7"
    >
      <Styled(LinkComponent)
        href="http://www.Denesik.ca/"
      >
        Corporis est quos nulla iste deleniti aut autem. Nulla ut impedit laudantium accusamus magni quia assumenda. Officiis cumque dolore quidem sunt et at earum molestiae quae. Qui accusamus architecto maiores dolorum ea qui quia voluptas. Libero quo sequi rerum labore quod aut sint.
      </Styled(LinkComponent)>
    </li>
    <li
      key="8HtSz7sNRyV"
    >
      <Styled(LinkComponent)
        href="http://Macejkovic.ca/"
      >
        Tenetur ratione et adipisci maxime doloribus suscipit molestiae. Perspiciatis aut sed. Voluptates eligendi quidem eos fugiat voluptates omnis tenetur aut. Illo placeat et et esse eos inventore. Corporis iusto omnis maxime est earum sint deleniti.
      </Styled(LinkComponent)>
    </li>
    <li
      key="n8VfvaqmwTZ"
    >
      <Styled(LinkComponent)
        href="http://Josefina.biz/"
      >
        Ratione sed sit distinctio nobis molestias doloremque. Ad sit nemo. Porro et sed aut quia explicabo pariatur. Beatae possimus porro iste ad.
      </Styled(LinkComponent)>
    </li>
  </StyledUl>
</StyledDiv>
`;
